core/foundation/test/testStringUtils.cxx

```diff
@@ -20,4 +20,23 @@ TEST(StringUtils, Split)
    test(",,,", {}, true);
```

Comment:
You are practically there, instead of putting your new test inside the scope of `TEST(StringUtils, Split)`, you can create a new test function that looks like

```cpp
TEST(StringUtils, Join)
{
    // test body
}
```